### PR TITLE
fix(ar): solve android crash for recognise image

### PR DESCRIPTION
**Pull request by ardasnturk**
- added `orientation` prop to `createTargets`
  because image recognition feature is crash
  in android
  -  orientation prop: determines the orientation
      of the source image

[viro document for orientation](https://viro-community.readme.io/docs/viroartrackingtargets#orientation)

SVA-633

### CODE REVIEW

File: src/components/augmentedReality/AugmentedRealityView.js [hunk 1]
```diff
@@ -31,6 +31,7 @@ export const AugmentedRealityView = ({ sceneNavigator }) => {
   useEffect(() => {
     ViroARTrackingTargets.createTargets({
       [TARGET]: {
+        orientation: 'Up',
```
Comment (donni106) [inline src/components/augmentedReality/AugmentedRealityView.js:34]: sure we need this? 'Up' seems to be the default: https://github.com/ViroCommunity/viro/blob/7fa83641c807f3065291d0d3a29bfef4cacabaa9/ios/ViroReact/AR/Modules/VRTARTrackingTargetsModule.mm#L284

Comment (donni106) [reply src/components/augmentedReality/AugmentedRealityView.js:34]: if this is the correct file, i cannot find a default for android: https://github.com/ViroCommunity/viro/blob/7fa83641c807f3065291d0d3a29bfef4cacabaa9/android/viro_bridge/src/main/java/com/viromedia/bridge/module/ARTrackingTargetsModule.java

**Review by donni106 — APPROVED** [review]

File: src/components/augmentedReality/AugmentedRealityView.js (hunk 1 above)
Comment (ardasnturk) [reply src/components/augmentedReality/AugmentedRealityView.js:34]: When I go to the augmented reality page on Android, it says that the `orientation` prop is missing and crash the application.

Comment (ardasnturk) [reply src/components/augmentedReality/AugmentedRealityView.js:34]: I think it is assigned as default data only for iOS. We can write this on discord.

Comment (donni106) [reply src/components/augmentedReality/AugmentedRealityView.js:34]: yes please share it with the community, it is something weird having a crash in this situation.

**Pull request merged**